Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -108,16 +113,18 @@
               app:layout_constraintTop_toTopOf="parent"
               tools:src="@drawable/avatar_3" />
 
-          <ImageView
-              android:id="@+id/email_star_image_view"
+          <com.google.android.material.button.MaterialButton
+              android:id="@+id/email_star_button"
+              style="@style/Widget.Material3.Button.IconButton"
               android:layout_width="@dimen/email_star_image_size"
               android:layout_height="@dimen/email_star_image_size"
-              android:padding="@dimen/grid_1"
-              android:background="?attr/selectableItemBackgroundBorderless"
-              android:contentDescription="@{email.starred ? @string/email_starred_content_desc : @string/email_unstarred_content_desc}"
-              android:foreground="@drawable/ic_circle"
-              android:src="@{email.starred ? @drawable/ic_star_filled : @drawable/ic_star_border}"
-              app:glideCircularCrop="@{true}"
+              android:checked="@{email.isStarred}"
+              android:checkable="true"
+              android:onClick="@{(view) -> email.toggleStar()}"
+              app:icon="@drawable/ic_toggle_star"
+              app:iconGravity="textStart"
+              app:iconTint="@color/star_statelist"
+              app:iconSize="@dimen/email_star_image_icon_size"
               app:layout_constraintEnd_toEndOf="parent"
               app:layout_constraintTop_toTopOf="parent" />
 
```

Comment:
Can you remove the `body_text_view`'s textColor attr on line 166? It's set to black. Must have missed this in a previous PR sorry!

I don't think a textColor needs to be set at all on that TextView